cookbooks/nginx/templates/default/nginx_app.conf.erb

```diff
@@ -82,12 +82,18 @@ server {
   #
   # Log files are stored within the /var/log/engineyard/nginx/ directory.
   #
+  <% log_prefix = if @syslog_enabled
+       'syslog:server=unix:'
```

Comment:
@wildbillcat According to the nginx docs (http://nginx.org/en/docs/syslog.html), the `server` config should point to the unix socket on which syslog listens on. In our case `dev/log`. In the current implementation it would expect the syslog socket to be the original log file and it would fail to open it as a socket.
Please change the configuration so it logs to the `/dev/log` socket in the case syslog is enabled.